Sources/IB.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  IB.swift
+//  IBAnimatableApp
+//
+//  Created by Tom Baranes on 17/01/2018.
+//  Copyright © 2018 IBAnimatable. All rights reserved.
+//
+
+import Foundation
+
+private struct AssociatedKeys {
+  static var ib = "IBAnimatable"
+}
+
+public final class IB: NSObject, CornerDesignable {
```

Comment:
I don't quite get it, why do we have two `IB` classes?

Reply:
Rebase failure 😔